### DISCUSSION TITLE
Map button to keyboard: how does it work ?

### DISCUSSION THREAD

**Discussion by jongena** [hneemann/Digital#990] [Q&A] [answered]
In the button specifications, there is the option "Map to Keyboard".
I guess that this is to create a visual keyboard, where each button has a character value that is automatically mapped to the keyboard component ? 
Or what is the purpose ? 
Can an example be given of its use ?

**Comment by hneemann** [comment, marked as answer]
The help text is actually not very helpful.
For example, if the button is named "T", the "T" key will be associated with this button during the simulation, so that pressing the "T" key on the keyboard will also press the corresponding button in the simulation.

**Reply by jongena** [reply]
Many thanks for the reply ! This explains !!
